src/utils/consent.js

```diff
@@ -0,0 +1,25 @@
+import { appVersion, sendEvent, getAppLanguage } from 'utils'
+
+const KEY = 'usage-data-consent'
+
+export const isConsentGranted = () => {
+  return localStorage.getItem(KEY) !== null
+}
+
+export const grantConsent = () => {
+  const version = appVersion()
+  sendEvent('KaiOSAppConsent', 20122102, getAppLanguage(), { version })
+  localStorage.setItem(KEY, JSON.stringify({
+    timestamp: Date.now(),
+    version: appVersion()
+  }))
+}
+
+// Those messages are not in i18n/*.json because they have a legal
+// value and are translated by legal experts. (more languages to come)
```

Comment:
I see, so as of right now: do we know if these message will eventually move to `i18n/*` or are they to stay in this util for the foreseeable future? How would the translation work, we add them directly here once we obtained them from legal experts?

Reply:
They should never be in `i18n/`. We'll receive professional translation for our target languages soon.

I think when it grows I will want to move it somewhere else so it's in its own file and easier to find.